profiles/winbind/README

```diff
@@ -48,6 +48,12 @@ with-pam-u2f-2fa::
 with-krb5::
     Enable Kerberos authentication with *pam_winbind*.
 
+with-winbind-cache::
+    Enable authentication cache with *pam_winbind*.
```

Comment:
I would write:

Enable logon using cached credentials with *pam_winbind* when 'winbind offline logon' is enabled.